src/main/java/edu/kis/vh/nursery/stack/IntArrayStack.java

```diff
@@ -13,15 +13,15 @@ public int getTotal() {
 
 	public void push(final int in) {
 		if (!isFull())
-			numbers[++total] = in;
+			numbers[total++] = in;
 	}
 
 	public boolean isEmpty() {
 		return total == Stack.IS_EMPTY_INDICATOR;
 	}
 
 	public boolean isFull() {
-		return total == CAPACITY-1;
+		return total == CAPACITY;
```

Comment:
Changing IS_EMPTY_INDICATOR value requires such fixes? It does not make sense. Incorrect fix.